libs/tlv-derive/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "tlv-derive"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
```

Comment:
I learned why `serde_derive` is its own crate and not part of `serde`. If your crate has procedural macros, then that's all it can be. You can't mix proc macro exports with non-proc macro exports. See https://doc.rust-lang.org/reference/procedural-macros.html.